File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int maxWordCount = 0;
        for (int i = 0; i < sentences.length; i++) {
            int spaceCount = 1;
            char[] arr = sentences[i].toCharArray();
            for (char c : arr) {
                if (c == ' ') {
                    spaceCount++;
                }
            }
            maxWordCount = Math.max(maxWordCount, spaceCount);
        }
        return maxWordCount;
    }
}

File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.cs
public class Solution {
    public int MostWordsFound(string[] sentences)
        {
            int wordCount = 1;
            int maxWordCount = 1;
            for (int i = 0; i < sentences.Length; i++)
            {
                wordCount = 1;
                char[] arr = sentences[i].ToCharArray();
                foreach (char c in arr)
                {
                    
                    if (c == ' ')
                    {
                        wordCount++;
                    }
                }
                maxWordCount = Math.Max(wordCount, maxWordCount);
            }
            return maxWordCount;
        }
}